src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkscanExceptionJourneyForAllClassificationTest.java

```diff
@@ -12,7 +12,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static uk.gov.hmcts.reform.bulkscan.endtoendtests.utils.ProcessorEnvelopeStatusChecker.getZipFileStatus;
 
-public class ExceptionRecordTest {
+public class BulkscanExceptionJourneyForAllClassificationTest {
```

Comment:
ExceptionJourney is not correct here as you have all classifications

Reply:
how about `BulkscanExceptionActionForAllClassificationTest`?

Reply:
it will mean test is for "All" Classification probably better to check on our team slack ?
I would go with `BulkscanWithJourneyClassificationsResultingInExceptionRecord`